bot/webserver.js

```diff
@@ -50,7 +46,7 @@ module.exports.startAPIServer = () => {
     app.post("/api", webAPI.onPost);
 };
 
-if (credentials.key === undefined || credentials.certificate === undefined) {
+if (appConfig.privateKey === "" || appConfig.certificate === "") {
```

Comment:
Might make more sense to use !appConfig.blalblaba? In case it's not defined in the json file for REASONS...

Reply:
Even better would be to use typeof. Even it should not be not defined if the user uses the install script, I'll change it